u256.go

```diff
@@ -233,7 +235,10 @@ func (u u256) div256by128(v u128) u128 {
 		a[i+1], a[i] = rem.hi, rem.lo
 	}
 
-	return u128{hi: q[1], lo: q[0]}
+	//nolint:gosec
+	r := u128{hi: a[1], lo: a[0]}.Rsh(uint(n))
```

Comment:
I assume the `nolint:gosec` is about converting `n` to uint.

I would add a comment explaining why it's OK

I tend to use nolintlint linter with `require - explanation:true`

Reply:
It's straightforward that n=bits.LeadingZero64 can be safely converted to uint, isn't it? I agree that nolint with an explanation is useful, tho. Will add it in a different PR.